src/coreclr/jit/lsra.h

```diff
@@ -771,7 +771,8 @@ class LinearScan : public LinearScanInterface
         LSRA_LIMIT_SMALL_SET = 0x3,
 #if defined(TARGET_AMD64)
         LSRA_LIMIT_UPPER_SIMD_SET = 0x2000,
-        LSRA_LIMIT_MASK           = 0x2003
+        LSRA_LIMIT_EXT_GPR_SET    = 0x4000,
+        LSRA_LIMIT_MASK           = 0x6003
```

Comment:
Did you verify if this gets automatically picked up with the `JitStressRegs` value we pass in CI? One of the place to see that list can be in: https://github.com/dotnet/runtime/blob/05d94d94028b5622b19734e0e1b60d7aca4667b3/src/coreclr/scripts/superpmi_replay.py#L30-L38

Reply:
I have checked this setting locally. But didn't know it needs to be enabled on CI. Will add and check

Reply:
https://github.com/dotnet/runtime/pull/108799#issuecomment-2635479404